internal/e2ebenchmark/tester/game/sys/jsys.go

```diff
@@ -0,0 +1,182 @@
+package sys
+
+import (
+	cryptorand "crypto/rand"
+	"math/big"
+
+	"github.com/argus-labs/world-engine/example/tester_benchmark/comp"
+	"pkg.world.dev/world-engine/cardinal"
+	"pkg.world.dev/world-engine/cardinal/types/entity"
+)
+
+var TenThousandEntityIds = []entity.ID{}
+var OneHundredEntityIds = []entity.ID{}
+var TreeEntityIds = []entity.ID{}
+
+func InitTenThousandEntities(wCtx cardinal.WorldContext) error {
+	var err error
+	entityAmount := 10000
+	TenThousandEntityIds, err = cardinal.CreateMany(wCtx, entityAmount, &comp.SingleNumber{Number: 1})
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func InitOneHundredEntities(wCtx cardinal.WorldContext) error {
+	var err error
+	entityAmount := 100
+	OneHundredEntityIds, err = cardinal.CreateMany(wCtx, entityAmount, &comp.ArrayComp{Numbers: [10000]int{}})
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func InitTreeEntities(wCtx cardinal.WorldContext) error {
+	var err error
+	var entityAmount = 100
+	var treeDepth = 10
+	wCtx.Logger().Info().Msg("CREATING tree entity")
+	tree := comp.CreateTree(treeDepth)
+	TreeEntityIds, err = cardinal.CreateMany(wCtx, entityAmount, *tree)
+	if err != nil {
+		wCtx.Logger().Info().Msg("ERROR CREATING tree entity")
+		wCtx.Logger().Info().Msg(err.Error())
+		return err
+	}
+	return nil
+}
+
+func SystemA(wCtx cardinal.WorldContext) error {
```

Comment:
It would be really nice if we could name the systems after what they do here, or at least just leave a short comment above each one. Much easier to read thru them that way since they are just named after a single letter currently.

Reply:
These are mostly Jeremies requested tests. He called it System A and B in his doc. Yeah I can change this.